ciphers/porta_cipher.py

```diff
@@ -0,0 +1,113 @@
+    "O": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P", "Q", "R", "S"]], "P": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P", "Q", "R", "S"]],
+    "Q": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P", "Q", "R"]], "R": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P", "Q", "R"]],
+    "S": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P", "Q"]], "T": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P", "Q"]],
+    "U": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P"]], "V": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P"]],
+    "W": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O"]], "X": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O"]],
+    "Y": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N"]], "Z": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N"]]}
+
+
+def generateTable(key):
+    """
+    >>> generateTable('marvin')
+    [[['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M'], ['U', 'V', 'W', 'X', 'Y', 'Z', 'N', 'O', 'P', 'Q', 'R', 'S', 'T']], [['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M'], ['N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z']], [['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M'], ['S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', 'N', 'O', 'P', 'Q', 'R']], [['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M'], ['Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', 'N', 'O', 'P']], [['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M'], ['W', 'X', 'Y', 'Z', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V']], [['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M'], ['U', 'V', 'W', 'X', 'Y', 'Z', 'N', 'O', 'P', 'Q', 'R', 'S', 'T']]]
+    """
+    table = []
+    for char in key.upper():
+        table.append(alphabet[char])
+    return table
```

Comment:
Please convert these four lines into a one line list comprehension.
`return [alphabet[char] for char in key.upper()]`